src/d3a/models/strategy/load_hours.py

```diff
@@ -19,13 +19,13 @@
 from pendulum import duration
 from typing import Union
 from collections import namedtuple
-
+from datetime import timedelta
```

Comment:
I know, it is in the update_frequency as well, but please use pendulum for managing times and durations (here you could use `from pendulum import duration`. 
At some point we decided to use pendulum instead of datetime. Thanks!